# result: compile error
def done = false
while (!done) {
  class Foo {
    method {
      break # expect: test:6: Error at 'break': Cannot use 'break' outside of a loop.
    }
  }
  done = true
}
